contrib/completion/zsh/_docker

```diff
@@ -2122,7 +2122,7 @@ __docker_subcommand() {
                 "($help)--ipv6[Enable IPv6 networking]" \
                 "($help -l --log-level)"{-l=,--log-level=}"[Logging level]:level:(debug info warn error fatal)" \
                 "($help)*--label=[Key=value labels]:label: " \
-                "($help)--live-restore[Enable live restore of docker when containers are still running]" \
+                "($help)--live-restore[Enable live restore of docker when containers are still running. This option is not supported on Windows.]" \
```

Comment:
This completion script will only be used if the client is on Linux, and the daemon on Windows. Wondering if we should take that into account, or just limit to the documentation

@albers @sdurrheimer wdyt?